python/paddle/fleet/__init__.py

```diff
@@ -14,10 +14,29 @@
 
 # TODO: define distributed api under this directory, 
 from .base.distributed_strategy import DistributedStrategy
-#from .base.role_maker import PaddleCloudRoleMaker, UserDefinedRoleMaker
-#from .base.fleet_base import Fleet
+from .base.fleet_base import Fleet
+from .base.util_factory import UtilBase
 
-#__all__ = [
-#    "DistributedStrategy", "PaddleCloudRoleMaker", "UserDefinedRoleMaker"
-#]
-__all__ = ['DistributedStrategy']
+#from .base.role_maker import PaddleCloudRoleMaker
+
+__all__ = ["DistributedStrategy", "UtilBase"]
+
+fleet = Fleet()
+init = fleet.init
+is_first_worker = fleet.is_first_worker
+worker_index = fleet.worker_index
+worker_num = fleet.worker_num
+is_worker = fleet.is_worker
+worker_endpoints = fleet.worker_endpoints
+server_num = fleet.server_num
+server_index = fleet.server_index
+server_endpoints = fleet.server_endpoints
+is_server = fleet.is_server
+util = fleet.util
+barrier_worker = fleet.barrier_worker
+init_worker = fleet.init_worker
+init_server = fleet.init_server
+run_server = fleet.run_server
+stop_worker = fleet.stop_worker
+distributed_optimizer = fleet.distributed_optimizer
+minimize = fleet.minimize
```

Comment:
If one call from `paddle.fleet import *`, may these assignment pollute namespace?

Reply:
I see this usage is something like horovod

Reply:
I thought another way to transmit those interfaces of Fleet instance is defining a `get_instance()` function:
```
def instance():
    return fleet
```
and invoking member functions from outside by
```
fleet.instance().minimize()
```
which may be flexible to synchronize functions in Fleet instance

Reply:
Just a suggestion

Reply:
sure, will change this to singleton in following updates. thx for ur advice